packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -54,7 +54,7 @@ export class Tasks extends React.Component {
               const TaskComponent = getTaskComponent(task.type)
               if (TaskComponent) {
                 return (
-                  <Box key={task.taskKey} basis='246px'>
+                  <Box key={task.taskKey} basis='auto'>
```

Comment:
This change is causing a style issue:

On this branch:
<img width="1366" alt="Screen Shot 2019-07-01 at 10 54 36 AM" src="https://user-images.githubusercontent.com/5016731/60450135-cf108980-9bee-11e9-95a2-ba13c769f339.png">

On production:
<img width="1360" alt="Screen Shot 2019-07-01 at 10 54 47 AM" src="https://user-images.githubusercontent.com/5016731/60450149-d59f0100-9bee-11e9-861a-ba57250c0767.png">

So the top spacing between the buttons below the classifier are depending on a min-height of the task area. The buttons should be managing their own margins.

Reply:
I think this has been fixed with the changes to TaskArea and adding `fill` or `flex` on related components.

Reply:
@srallen I might be mistaken but I think your "requested changes" status is related to this styling issue? if so, is this looking ok now? @simensta and I are looking to get this PR cleaned up and ready for merge early this week.

Reply:
<img width="589" alt="Screen Shot 2019-07-29 at 1 12 35 PM" src="https://user-images.githubusercontent.com/5016731/62071624-93270f00-b202-11e9-95c3-d1f3249d165d.png">

This still is showing the metadata buttons right up against the subject viewer. As long as we note this and do a follow up PR addressing this, I can dismiss my review and allow @simensta to finish review for merging.